examples/window_resize_increments.rs

```diff
@@ -0,0 +1,68 @@
+#[cfg(any(target_os = "macos", all(target_os = "linux", feature = "x11")))]
+fn main() {
+    use simple_logger::SimpleLogger;
+    use winit::{
+        dpi::LogicalSize,
+        event::{ElementState, Event, KeyboardInput, VirtualKeyCode, WindowEvent},
+        event_loop::EventLoop,
+        window::WindowBuilder,
+    };
+
+    #[cfg(target_os = "macos")]
+    use winit::platform::macos::WindowBuilderExtMacOS;
+
+    #[cfg(all(target_os = "linux", feature = "x11"))]
+    use winit::platform::unix::WindowBuilderExtUnix;
+
+    SimpleLogger::new().init().unwrap();
+    let event_loop = EventLoop::new();
+
+    let window = WindowBuilder::new()
+        .with_title("A fantastic window!")
+        .with_inner_size(LogicalSize::new(128.0, 128.0))
+        .with_resize_increments(LogicalSize::new(25.0, 25.0))
+        .build(&event_loop)
+        .unwrap();
+
+    let mut has_increments = true;
+
+    event_loop.run(move |event, _, control_flow| {
+        control_flow.set_wait();
+
+        match event {
+            Event::WindowEvent {
+                event: WindowEvent::CloseRequested,
+                window_id,
+            } if window_id == window.id() => control_flow.set_exit(),
+            Event::WindowEvent {
+                event:
+                    WindowEvent::KeyboardInput {
+                        input:
+                            KeyboardInput {
+                                virtual_keycode: Some(VirtualKeyCode::Space),
+                                state: ElementState::Released,
+                                ..
+                            },
+                        ..
+                    },
+                window_id,
+            } if window_id == window.id() => {
+                has_increments = !has_increments;
+                log::debug!("Has increments: {}", has_increments);
```

Comment:
Could just `use log::debug`.

Reply:
Ok that's a personal habit, you can always write `log::something!("msg")` without importing anything and semantically it looks nice. I'll import it here tho